README.md

```diff
@@ -30,6 +30,38 @@ kubectl logs -f $(kubectl get pods --namespace default -l "run=cleanup-operator"
 kubectl create -f https://k8s.io/examples/controllers/job.yaml
 ```
 
+### Parameters
```

Comment:
Parameters specific to the cleanup-operator could be found at the bottom of the Readme, in the development section.
Do you think moving it to the separate section improves readability? If so, please remove it from the development section instead of the duplication.